mobile/lib/shared/services/network.service.dart

```diff
@@ -109,7 +109,7 @@ class NetworkService {
 
       String validUrl = Uri.parse('$savedEndpoint/server-info/ping').toString();
 
-      debugPrint("pint server at url $validUrl");

```

Comment:
I was drunk when typing this, with a pint in my left hand and the keyboard in my right hand :trollface:

Reply:
Understandable hahaha, no worries 😄